includes/Admin/Settings_Screens/Messenger.php

```diff
@@ -268,56 +255,34 @@ public function render() {
 	 * @throws Framework\SV_WC_Plugin_Exception
 	 */
 	public function save() {
-
-		$plugin               = facebook_for_woocommerce();
-		$external_business_id = $plugin->get_connection_handler()->get_external_business_id();
-
-		try {
-
-			// first get the latest configuration details
-			$response = $plugin->get_api()->get_business_configuration( $external_business_id );
-
-			$configuration = $response->get_messenger_configuration();
-
-			if ( ! $configuration ) {
-				throw new Framework\SV_WC_API_Exception( 'Could not retrieve latest messenger configuration' );
-			}
-
+		// first get the latest configuration details
+		$external_business_id = facebook_for_woocommerce()->get_connection_handler()->get_external_business_id();
+		$configuration        = facebook_for_woocommerce()->get_integration()->get_messenger_configuration( $external_business_id );
+		if ( $configuration ) {
 			$update          = false;
 			$setting_enabled = wc_string_to_bool( Framework\SV_WC_Helper::get_posted_value( \WC_Facebookcommerce_Integration::SETTING_ENABLE_MESSENGER ) );
```

Comment:
Did we miss this or the use of SV_WC_Helper::get_posted_value will be phased out with a later PR?